stories/Scrollable.stories.js

```diff
@@ -38,3 +37,60 @@ stories.add('Default', () => (
     </Scrollable>
   </Card>
 ))
+
+const MainContainer = styled('div')`
+  ${baseStyles} width: ${props => (props.fullWidth ? '100%' : '300px')};
```

Comment:
@tinkertrain OoO! I found that adding a `;` at the end of things like:

```
${baseStyles}
```

helps with Prettier indentation 👍

Reply:
That drove me crazy, thanks!